sdk/databoxedge/Azure.ResourceManager.DataBoxEdge/src/autorest.md

```diff
@@ -105,6 +105,7 @@ rename-mapping:
   ClusterStorageViewData.clusterFreeStorageMb: ClusterFreeStorageInMB
   ClusterStorageViewData.clusterTotalStorageMb: ClusterTotalStorageInMB
   GenerateCertResponse: GenerateCertResult
+  GenerateCertResponse.expiryTimeInUTC: ExpireOn|datetime
```

Comment:
Can we use directive instead of rename-mapping to update the format? Just like what I found in this [issue comment](https://github.com/Azure/autorest.csharp/issues/2655#issuecomment-1250582356), the rename mapping will not update the serialization format.